val.py

```diff
@@ -441,36 +453,88 @@ def run(
                 save_one_json(predn, jdict, path, class_map)  # append to COCO-JSON dictionary
             callbacks.run('on_val_image_end', pred, predn, path, names, im[si])
 
-            # TODO the following code contains a bug and will be fixed in PR #13
-            pred_clone[:, :4] = scale_boxes(im.shape[2:], pred_clone[:, :4],
-                                            im0[si].shape).round()
+            if skip_evaluation or save_blurred_image:
```

Comment:
can you explain this condition? why is it or?

Reply:
Im running the evaluation with save-blurred-images set to True and this is executed

Reply:
save-blurred-images -> --save-blurred-image, without the s.

I also find the or confusing... I will change it to be skip_evaluation inside the save_blurred_image if statement, at the database insertion part.